scripts/scr_weapon/scr_weapon.gml

```diff
@@ -1681,6 +1542,166 @@ global.weapons = {
 		"arp": 0,
 		"tags": ["vehicle", "turret", "volkite", "ancient"]
 	},
+// Hireling weapons
+	// Admech
+	"Hellgun": {
+		"abbreviation": "HllGun",
+		"attack": {
+			"standard": 30,
+			"master_crafted": 34,
+			"artifact": 38
+		},
+		"ammo": 10,
+		"range": 6.1,
+		"spli": 3,
+		"arp": 0,
+		"tags": [],
+		"ranged_hands": 1,
+		// "tags": ["arcane"], - how is it supposed to be arcane?
+		// ... (other attributes)
+	},
+	"Laspistol": {
+		"attack": {
+			"standard": 20,
+			"master_crafted": 30,
+			"artifact": 40
+		},
+		"description": "The Laspistol is the pistol version of the Lasgun and like that weapon fires a coherent beam of energetic photons that can burn through most materials. The Laspistol is powered by a miniature power pack that is usually placed within the grip.",
+		"abbreviation": "Lpstl",
+		"melee_hands": 0,
+		"ranged_hands": 0.25,
+		"ammo": 30,
+		"range": 3.1,
+		"spli": 0,
+		"arp": 0,
+		"tags": ["pistol", "las"],
+	},
+	// Other imperials
+	"Light Bolter": {
+		"abbreviation": "LiBltr",
+		"attack": {
+			"standard": 40,
+			"master_crafted": 45,
+			"artifact": 50
+		},
+		"description": "A smaller variant of Bolter, intended to be useable by unaugmented humans.",
+		"melee_hands": 0,
+		"ranged_hands": 1,
+		"ammo": 16,
+		"range": 12,
+		"spli": 3,
+		"arp": 0,
+		"tags": ["bolt"]
+	},
```

Comment:
Nerfed this one a little bit more, again, according to various sources and our current weapons.
Also added Sarissa as a built-in weapon, as it's a bayonet.
```suggestion
	"Light Bolter": {
		"abbreviation": "LghtBltr",
		"attack": {
			"standard": 35,
			"master_crafted": 40,
			"artifact": 45
		},
		"description": "A smaller variant of Bolter, intended to be useable by unaugmented humans.",
		"melee_hands": 0,
		"ranged_hands": 1,
		"ammo": 16,
		"range": 10,
		"spli": 2,
		"arp": 0,
		"tags": ["bolt"],
		"second_profiles": ["Sarissa"]
	},
```